docs/services/sdk/overview.md

```diff
@@ -18,8 +18,50 @@ Add the `@restatedev/restate-sdk` dependency to your NodeJS project to start dev
 The Typescript SDK uses the `protobufjs` and `ts-proto` dependencies to work with the Protobuf-generated Typescript code.
 So add these as well to your project.
```

Comment:
Wondering whether service types should come before the API since they more or less speak primarily about the Protobuf contract which is not relevant for the handler API.

Reply:
Service types stands above API/overview in the docs, so I think this is already the case